pkg/controller/jobs/pod/pod_controller.go

```diff
@@ -427,6 +427,10 @@ func (p *Pod) GVK() schema.GroupVersionKind {
 	return gvk
 }
 
+func (p *Pod) PodLabelSelector() string {
```

Comment:
> Please share how the command for pod groups looks like? Is it something like:
> 
> `kueuectl list pods --for pod/{workload-name}`?

Yes, we have like this.

> That looks odd. Maybe this will need to be specialized to look like this:
> 
> `kueuectl list pods --for podgroup/{workload-name}`?

I agree. It looks better.

Reply:
Another alternative would be:

```
kueuectl list pods --for workload/{workload-name}
```

But then it might be a bit more effort to get it to work for any kind of workload, and not just those that match a pod group.

Or:


```
kueuectl list pods --for pod/{pod-name}
```

which should give you all the pods that belong to the same group.

But, we could leave pod groups unsupported for now and add it later, so that we don't delay this much longer.

Reply:
Ahh, sorry, my mistake, I mean that we have this solution

```
kueuectl list pods --for pod/{pod-name}
```